src/workspacer.Bar/Widgets/TextWidget.cs

```diff
@@ -18,7 +18,7 @@ public TextWidget(string text)
 
         public override IBarWidgetPart[] GetParts()
         {
-            return Parts(Part(_text,null,null,null,FontName));
+            return Parts(Part(_text, fore: null, back: null, partClicked: null, FontName));
```

Comment:
IN this case I was more thinking eliminating redundant parameters using named arguments. Like this:

````c#
return Parts(Part(_text, fontName: FontName));
````

Reply:
oh my bad, totally misunderstood you there